pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotQueryResource.java

```diff
@@ -66,8 +72,31 @@
   @Inject
   AccessControlFactory _accessControlFactory;
 
+  @POST
+  @Path("queryByFormat")
+  public String postQuery(String requestJsonStr, @Context HttpHeaders httpHeaders) {
+    try {
+      JsonNode requestJson = JsonUtils.stringToJsonNode(requestJsonStr);
+      String query = requestJson.get("query").asText();
+      String traceEnabled = "false";
+      if (requestJson.has("trace")) {
```

Comment:
Minor nit: This can be a one-liner like:
`String traceEnabled = (requestJson.has("trace")) ? .... : false;`